### PR TITLE
Edits from Adam

### PR DESCRIPTION
**Pull request by adamkrellenstein**

### CODE REVIEW

File: src/content/header/header.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: The OG tokenization protocol on Bitcoin
+title: The Original Bitcoin Layer-2 Protocol
```
Comment (pinnate-modo) [inline src/content/header/header.md:2]: Maybe "Original DeFi and NFT Protocol on Bitcoin"?

File: src/content/marketplaces/marketplaces.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Marketplaces
 introduction: >
-  These marketplaces let you buy and trade NFTs of the Counterparty ecosystem in
+  These marketplaces let you buy and trade NFTs on the Counterparty network in
```
Comment (pinnate-modo) [inline src/content/marketplaces/marketplaces.md:4]: NACK. This is precisely what they don't let you do. Original wording is better.

**Review by pinnate-modo — CHANGES_REQUESTED** [review]
Once my comments are addressed good to go!

File: src/content/header/header.md (hunk 1 above)
Comment (ffmad) [reply src/content/header/header.md:2]: Layer 2 is in contradiction with Protocol

**Pull request merged**